contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+# LBRY on Kubernetes with Helm
```

Comment:
One of the things that I would love for the documentation is to avoid making assumptions of knowledge in general. So when a user jumps into wanting to host their own K8's based instance of this we should assume that the user will want links back to the official documentation for various processes.

Reply:
added